src/arkode/arkode_impl.h

```diff
@@ -697,6 +698,10 @@ SUNErrCode arkSUNStepperSelfDestruct(SUNStepper stepper);
 int arkSetForcePass(void* arkode_mem, sunbooleantype force_pass);
 int arkGetLastKFlag(void* arkode_mem, int* last_kflag);
 
+/* Utility function to tell ARKode to free the user data.
+   This is used by the Python interfaces. */
+int ARKodeSetOwnUserData(void* ark_mem, sunbooleantype own_user_data);
```

Comment:
Prefix internal function with `ark`
```suggestion
int arkSetOwnUserData(void* ark_mem, sunbooleantype own_user_data);
```